schema/system_hours.json

```diff
@@ -0,0 +1,102 @@
+{
+  "description": "Describes the hours of operation for the system.",
+  "definitions": {
+    "RentalHours": {
+      "description": "An hour object",
+      "type": [
+        "object",
+        "null"
+      ],
+      "properties": {
+        "user_types": {
+          "description": "An array of \"member\" and \"nonmember\" values. This indicates that this set of rental hours applies to either members or non-members only.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "member",
+              "nonmember"
+            ]
+          },
+          "minItems": 1,
+          "maxItems": 2
+        },
+        "days": {
+          "description": "An array of abbreviations (first 3 letters) of English names of the days of the week that this hour object applies to (i.e. [\"mon\", \"tue\"]). Each day can only appear once within all of the hours objects in this feed.",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "sun",
+              "mon",
+              "tue",
+              "wed",
+              "thu",
+              "fri",
+              "sat"
+            ]
+          },
+          "minItems": 1,
+          "maxItems": 7
+        },
+        "start_time": {
+          "description": "Start time for the hours of operation of the system in the time zone indicated in system_information (00:00:00 - 23:59:59)",
+          "type": "string",
+          "pattern": "([0-1][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9]"
+        },
+        "end_time": {
+          "description": "End time for the hours of operation of the system in the time zone indicated in system_information (00:00:00 - 47:59:59). Time can stretch up to one additional day in the future to accommodate situations where, for example, a system was open from 11:30pm - 11pm the next day (i.e. 23:30-47:00)",
```

Comment:
@thzinc I have a question regarding the `end_time` attribute value. Is it valid and accepted by the GBFS feeds consumers to have the `end_time` stretched up to one additional day in the future? This is the exact situation I am facing now and I just wanted to double check. Thanks.

Reply:
Yes, that's explicitly allowed by the Time definition in gbfs.md [Field Types](https://github.com/NABSA/gbfs/blob/master/gbfs.md#field-types):

Time - Service time in the HH:MM:SS format for the time zone indicated in system_information.json (00:00:00 - 47:59:59). Time can stretch up to one additional day in the future to accommodate situations where, for example, a system was open from 11:30pm - 11pm the next day (i.e. 23:30:00-47:00:00).

Reply:
@heidiguenin thank you